h/static/scripts/test/drafts-test.js

```diff
@@ -11,23 +11,25 @@ describe('drafts', function () {
     it('should save changes', function () {
       var model = {id: 'foo'};
       assert.notOk(drafts.get(model));
-      drafts.update(model, {text: 'edit'});
-      assert.deepEqual(drafts.get(model), {text: 'edit'});
```

Comment:
I actually think it was easier to work with the previous versions of the API methods where the parameters were passed via an object. `drafts.update(model, {isPrivate: true, ...})` seems clearer than `drafts.update(model, true, ...)` to me.